test/manual/diy-consent.html

```diff
@@ -1,25 +1,111 @@
-        I am glad you decided to learn more! Learning is great!
-      </p>
+
+  <head>
+    <link rel="stylesheet" type="text/css" href="https://raw.githack.com/ganapativs/bttn.css/master/dist/bttn.min.css">
+    <link rel="stylesheet" href="https://unpkg.com/sakura.css/css/sakura.css" type="text/css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/3.7.0/animate.min.css">
+    <style>
+html {
+  width: 100%;
+  background-color: #fff;
+  padding: 0px !important;
+}
+
+body {
+  width: 90%;
+  margin-left: auto;
+  margin-right: auto;
+  max-width: 800px;
+  background-color: #fff;
+  line-height: initial;
+  margin: 0;
+  padding: 5px;
+}
+
+.bttn-primary {
+  margin-right: 10px;
+}
+
+.decision-text {
+  font-size: 0.85em;
+  word-wrap: break-word;
+}
+
+h1 {
+  font-size: 1.15em;
+}
+
+#decision-buttons {
+  margin-top: 10px;
+  margin-bottom: 10px;
+}
+
+#always-show {
+  position: absolute;
+  top: 5px;
+  right: 10px;
+}
+
+.close {
+  transition: all 0.5s;
+  color: red;
+  font-size: 1em;
+  background-color: transparent;
+  border: 1px solid #000;
+  padding-left: 5px;
+  padding-right: 5px;
+  padding-top: 2px;
+  padding-bottom: 2px;
+}
+
+.close:hover {
+  color: red;
+  font-size: 1.25em;
+  background-color: transparent;
+  border: 1px solid #000;
+}
+    </style>
+  </head>
+
+  <body>
+
+    <div id="consent-wrapper">
+      <div id="decision-state">
+        <h1 class="animated fadeIn">We value your privacy!</h1>
+        <div class="decision-text animated fadeIn">
+          This website uses a technologies like Cookies or Targeting in order to personalize the advertising that you see. This helps us to show you more relevant ads and improves your internet experience. We also use it in order to measure results or align our website content. Because we value your privacy, we are here with asking for your permission to use this technologies.
+        </div>
+      </div>
+      <div id="learn-more-state">
+
+        <h1 class="animated fadeIn">More Information</h1>
+        <div class="animated fadeIn">
+          The General Data Protection Regulation (EU) 2016/679 ("GDPR") is a regulation in EU law on data protection and privacy for all individuals within the European Union (EU) and the European Economic Area (EEA). It also addresses the export of personal data outside the EU and EEA areas. The GDPR aims primarily to give control to individuals over their personal data and to simplify the regulatory environment for international business by unifying the regulation within the EU.[1] Superseding the Data Protection Directive 95/46/EC, the regulation contains provisions and requirements pertaining to the processing of personal data of individuals (formally called data subjects in the GDPR) inside the EEA, and applies to an enterprise established in the EEA or—regardless of its location and the data subjects' citizenship—that is processing the personal information of data subjects inside the EEA.
```

Comment:
Can we remove those and replace with some fake text. I would rather not be explicit incase the text itself is used as template.